README.md

```diff
@@ -239,11 +239,13 @@ By default, this cookbook installs Agent v6, if you want to install Agent v5, pl
 
 Attributes are available to have finer control over how you install Agent v6:
 
- * `agent6_version`: allows you to pin the agent version (recommended).
- * `agent6_package_action`: defaults to `'install'`, may be set to `'upgrade'` to automatically upgrade to latest (not recommended, we recommend pinning to a version with `agent6_version` and change that version to upgrade).
- * `agent6_aptrepo`: desired APT repo for the agent. Defaults to `http://apt.datadoghq.com`
- * `agent6_aptrepo_dist`: desired distribution for the APT repo. Defaults to `stable`
- * `agent6_yumrepo`: desired YUM repo for the agent. Defaults to `https://yum.datadoghq.com/stable/6/x86_64/`
+ * `agent_major_version`: allows you to pin the major release of the agent (recommended)
+ * `agent_version`: allows you to pin a specific agent version (recommended).
```

Comment:
The problem with the pinning of minor versions is that you have to specify the version as the package manager wants it (so, its different in deb/rpm/windows). I've tried to keep the changes to the minimum, so I only changed the boolean to a an int, while keeping the `agent_version` field as it was. I agree it's a bit confusing, but it's not worse than what we had.

Having a single version field doesn't let you pin the major version without pinning the minor and patch versions, which I think can be a use case. Maybe we could support stuff like "6.x", but I'm not sure if that's common practice.

Reply:
> I agree it's a bit confusing, but it's not worse than what we had.

Fair point, and again, it works well 👍 . My thoughts were about the parts of code where we have the conditions on `agent_major_version` to later use `agent_version` to do something (mainly in `repository.rb`). The value of these fields will be one of the first thing to check in case of errors.